evm-contracts/src/v0.6/dev/AggregatorProxy.sol

```diff
@@ -281,13 +327,47 @@ contract AggregatorProxy is AggregatorInterface, AggregatorV3Interface, Owned {
     setAggregator(_aggregator);
   }
 
+
   /*
    * Internal
    */
 
   function setAggregator(address _aggregator)
     internal
   {
+    epoch++;
+    epochAggregators[epoch] = AggregatorV3Interface(_aggregator);
     aggregator = AggregatorV3Interface(_aggregator);
   }
+
+
+  // PRIVATE
+
+  function addEpoch(
+    uint256 _epoch,
+    uint256 _originalId
+  )
+    private
+    view
+    returns (uint256)
+  {
+    return _epoch.mul(EPOCH_BASE).add(_originalId);
+  }
+
+  function parseRequestId(
+    uint256 requestId
+  )
+    private
+    view
+    returns (uint16, uint256)
```

Comment:
Could we use a `uintEPOCH_OFFSET` for the second return value? I feel that would communicate the semantics more clearly.